Team2 group skills

hinckleyc opened the pull request:
We added information to:

Deadlines
Participation
Conduct
Conflicts
Infractions

Please let me know if I need to change anything to improve?

koscinskic committed:
Added participation guidelines in assessment.md

All quality levels as well as an E-grade example for using appropriate commit messages were added

Commit:
Add 'deadlines' section to group skills

Commit:
added conflicts guidlines in assesment.md

TylerPham2000 committed:
Added conduct descriptions into group

Commit:
made changes to conflicts in assessment.md

Commit:
added infractions to the group skills

Commit:
made changes to infractions in assessment.md

enpuyou reviewed (changes requested):
@hinckleyc Please reference to https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown for the style of assessment. Also, there is an inconsistent use of bold texts. If your team believe that this will help with the readability, you may want to raise an issue for the style guide or contact other students on the markdown style guide team for the change

koscinskic committed:
Added emoji rating symbols

Converted the original NIAGE ratings to the new emoji ratings.

Corrected line length to accomadate this change.

koscinskic commented:
I have opened an issue on the Markdown team repository concerning the usage of bold text as well as resolved the requested changes concerning emoji style of assessment.

hodevin commented:
There are still some inconsistent use of period punctuation but other than that, it looks great.

corlettim commented on assessment.md:
```diff
@@ -81,15 +81,86 @@ Project Management is Covered by Team 2
+E level by completing their assigned work for the project with enough time
+remaining to properly test and document it, and enough time to fix any errors.
+Additionally, they should encourage others to complete their work in
+a similar time frame such that the project flows as smoothly as possible.
```
Should this still be here since you have now begun the use of emojis instead of the letters?

corlettim commented on assessment.md:
```diff
@@ -81,15 +81,86 @@ Project Management is Covered by Team 2
+E level by understanding their role / position within the group and handling
+aspects of the project that concern this role / position. Furthermore, they
+will effectively communicate their intentions and abilities in order to
+maximize productivity.
```
Same as above...this may not be needed

koscinskic replied:
I changed the "E" to the appropriate emoji substitute on both branches, but I am still under the impression that an example is warranted regardless of the symbolic representation. The suggestion for an example blurb is in the original README.md

koscinskic committed:
Changed E symbol to :sunglasses: in guideline examples

Commit:
Add communications

Commit:
Fixing communication skills

koscinskic committed:
Added Attendance in assessment.md

These guidelines were submitted via Slack as they are Thomas' material

noorbuchi commented:
Hi, I'm a member of the markdown team and I've been trying to review some of the pull requests and make sure that the branches adhere to the markdown style in the markdown repository . There is a small issue with your pull request in which you are using bold in your writing. Please let me know if you any questions about markdown usage. Thanks!

koscinskic commented:
After reviewing the Markdown repository once more, I have found no guidelines governing the usage of modified text. If team members should not be using bold text under any circumstances, please specify this in the Markdown repository. Otherwise, this is an invalid issue that should not prohibit the merging of this PR.

Commit:
Add Anthony's decisions section to group skills

Commit:
Fix trailing spaces

corlettim reviewed (approved):
this looks good!

koscinskic committed:
Removed bolded text in assessment.md

Changes made in adherence to Markdown guidelines

noorbuchi commented:
@koscinskic the markdown repository has recently been updated to demonstrate the fact bold should not be used. Additionally, there was a recent announcement notifying everyone of this change.

koscinskic commented:
All of the guidelines should be adhered to, thank you for the clarification.

enpuyou committed:
Merge branch 'master' into team2-group-skills

enpuyou committed:
Merge branch 'master' into team2-group-skills

enpuyou commented on assessment.md:
```diff
@@ -79,15 +79,110 @@
+:sunglasses: level by completing their assigned work for the project with enough
+time remaining to properly test and document it, and enough time to fix any
+errors. Additionally, they should encourage others to complete their work in a
+similar time frame such that the project flows as smoothly as possible.
```
This section needs to be removed according to the latest assessment style guide

enpuyou commented on assessment.md:
```diff
@@ -79,15 +79,110 @@
+:sunglasses: level by understanding their role / position within the group and
+handling aspects of the project that concern this role / position. Furthermore,
+they will effectively communicate their intentions and abilities in order to
+maximize productivity.
```
Same here

enpuyou reviewed (changes requested):
The period in each bullet point line needs to be removed

koscinskic committed:
Removed Concrete Explanations for Professional Guidelines

*HIGHLY UNADVISED ACTION*
Done in Compliance to Markdown Standard

ibarrezueta committed:
Merge branch 'master' into team2-group-skills

enpuyou committed:
Merge branch 'master' into team2-group-skills

enpuyou committed:
Remove unneeded black lines

gkapfham reviewed (approved):
Thanks for your work on this PR, I approve you to merge it to the master branch.
